common/protob/messages.proto

```diff
@@ -317,7 +319,24 @@ enum MessageType {
     MessageType_SolanaTxSignature = 905 [(wire_out) = true];
 
     // THP
-    reserved 1000 to 1099; // See messages-thp.proto
+    MessageType_ThpCreateNewSession = 1000 [(bitcoin_only) = true, (wire_in) = true];
+    MessageType_ThpPairingRequest = 1006 [(bitcoin_only) = true];
+    MessageType_ThpPairingRequestApproved = 1007 [(bitcoin_only) = true];
+    MessageType_ThpSelectMethod = 1008 [(bitcoin_only) = true];
+    MessageType_ThpPairingPreparationsFinished = 1009 [(bitcoin_only) = true];
+    MessageType_ThpCredentialRequest = 1010 [(bitcoin_only) = true];
+    MessageType_ThpCredentialResponse = 1011 [(bitcoin_only) = true];
+    MessageType_ThpEndRequest = 1012 [(bitcoin_only) = true];
+    MessageType_ThpEndResponse = 1013 [(bitcoin_only) = true];
+    MessageType_ThpCodeEntryCommitment = 1016 [(bitcoin_only) = true];
+    MessageType_ThpCodeEntryChallenge = 1017 [(bitcoin_only) = true];
+    MessageType_ThpCodeEntryCpaceTrezor = 1018 [(bitcoin_only) = true];
+    MessageType_ThpCodeEntryCpaceHostTag = 1019 [(bitcoin_only) = true];
+    MessageType_ThpCodeEntrySecret = 1020 [(bitcoin_only) = true];
+    MessageType_ThpQrCodeTag = 1024 [(bitcoin_only) = true];
+    MessageType_ThpQrCodeSecret = 1025 [(bitcoin_only) = true];
+    MessageType_ThpNfcTagHost = 1032 [(bitcoin_only) = true];
+    MessageType_ThpNfcTagTrezor = 1033 [(bitcoin_only) = true];
```

Comment:
tiny nit: if needed, it is possible to remove THP-related strings from non-THP builds (similar to https://github.com/trezor/trezor-firmware/pull/4949):
```
$ TREZOR_MODEL=T3B1 BITCOIN_ONLY=0 QUIET_MODE=1 make -C core build_firmware
$ strings core/build/firmware/firmware.bin | grep 'Thp'
ThpCodeEntryChallenge
ThpCodeEntryCommitment
ThpCodeEntryCpaceHostTag
ThpCodeEntryCpaceTrezor
ThpCodeEntrySecret
ThpCreateNewSession
ThpCredentialRequest
ThpCredentialResponse
ThpEndRequest
ThpEndResponse
ThpNfcTagHost
ThpNfcTagTrezor
ThpPairingPreparationsFinished
ThpPairingRequest
ThpPairingRequestApproved
ThpQrCodeSecret
ThpQrCodeTag
ThpSelectMethod
ThpUnallocatedSession
trezor/enums/ThpPairingMethod.py
trezor/enums/ThpPairingMethod.py
```